index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/a12691952c.js" crossorigin="anonymous"></script>
+    <title>Portfolio David Tamayo</title>
+</head>
+<body>
+    <div class="mainpage">
+        <header class="toolbar">
+            <h2 class="welcome">Welcome</h2>
+            <button class="vector">☰</button>
+        </header>
+        <section class="headline">
+            <h1 class="black">Hey there. I'm David</h1>
+            <h1 class="green">I'm a software developer</h1>
+        </section>
+        <article class="presentation">
+            <h3 class="content"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h3>
+        </article>
```

Comment:
- [ ] I would recommend you use  a `p` tag instead of `article` because this way will be semantically correct and the screen readers :smile: